src/main.rs

```diff
@@ -35,7 +39,7 @@ struct Options {
     ///
     /// This must be a crates.io package name.
     #[clap(value_name = "crate")]
-    crate_name: CrateName,
+    crate_names: Vec<CrateName>,
```

Comment:
About `Options`, I also think we should also reject if `manifest_path` is specified and there are multiple crates specified, but not sure the name of the `BinstallError` variant to add.

Reply:
@passcod Shall I add some checking here?